lib/web/servers.go

```diff
@@ -75,9 +80,10 @@ func (h *Handler) clusterDatabasesGet(w http.ResponseWriter, r *http.Request, p
 	}
 
 	return listResourcesGetResponse{
-		Items:      ui.MakeDatabases(h.auth.clusterName, databases),
-		StartKey:   resp.NextKey,
-		TotalCount: resp.TotalCount,
+		Items:                ui.MakeDatabases(h.auth.clusterName, databases),
+		StartKey:             resp.NextKey,
+		TotalCount:           resp.TotalCount,
+		AssumedRoleRequestID: assumedRoleRequestID,
```

Comment:
```suggestion
		AssumedRoleRequestID: ctx.session.GetAssumedRoleRequestID(),
```

And then get rid of the variable above. (No need for a var if you're only using it once)